docs/content/modeling/migrating/migrating-schema-1-1.mdx

```diff
@@ -0,0 +1,494 @@
+---
+sidebar_position: 2
+slug: /modeling/migrating/migrating-schema-1-1
+---
+
+import {
+  AuthzModelSnippetViewer,
+  CardBox,
+  CheckRequestViewer,
+  DocumentationNotice,
+  ProductConcept,
+  ProductName,
+  ProductNameFormat,
+  ReadRequestViewer,
+  RelatedSection,
+  RelationshipTuplesViewer,
+  WriteRequestViewer,
+} from '@components/Docs';
+
+# Migrating Models To Schema 1.1
+
+<DocumentationNotice />
+
+<ProductName format={ProductNameFormat.ShortForm} /> has introduced a new DSL schema version with several changes that we
+believe will make models easier to read and write, enable better tuple and model validations, and provide more options for
+optimizing the performance of different <ProductName format={ProductNameFormat.ShortForm} /> APIs.
+
+In short, we will be:
+
+1. [Adding model schema version field](#-model-schema-versions)
+2. [Adding type enforcements and removing need to specify `as self`](#type-enforcements--removing-as-self)
+3. [Disallowing string literals in user_ids](#disallowing-string-literals-in-user_ids)
+4. [Enforcing userset type restrictions](#enforcing-userset-type-restrictions)
+5. [Requiring you to specify for which relations you can write tuples with public access (using ‘`*`’)](#public-access)
+6. [Changes in query evaluation behavior with type restrictions](#query-evaluation-behavior-with-type-restrictions)
+
+To facilitate migration to the new DSL schema, you will need to update tuples that are no longer valid. In particular, all tuples with wildcard (`*` or `user:*`) user field defined with model schema 1.0 <u><strong>MUST</strong></u> be deleted and re-added back.
+
+## <ProductName format={ProductNameFormat.ShortForm}/> Model Schema Versions
+
+Since the changes in the DSL are significant, we have decided to add a schema version to the DSL. The previous version of the DSL’s schema was `1.0`, and the new schema version will be `1.1`. To use the new syntax please add the following to the top of the model:
+
+```
+model
+  schema 1.1
+```
+
+<ProductName format={ProductNameFormat.ShortForm} /> will eventually stop supporting schema version 1.0. Notifications will
```

Comment:
Please make this a callout instead of a regular paragraph.

Reply:
As discussed offline, admonition